### PR TITLE
Add estimate as nested attribute on travel request

### LINKED ISSUE #62: Cost Estimate
- Request
- Type
   - Get the type information from the previous system
- Amount
- Recurrence

### PR DESCRIPTION
**Pull request by hackmastera**
uses a postgres enum for the type of estimate cost. Note this means we must switch from using `schema.rb` to `structure.sql`.

closes #62

### CODE REVIEW

File: db/migrate/20190501192018_add_type_to_estimates.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class AddTypeToEstimates < ActiveRecord::Migration[5.2]
+  def up
+    execute <<-SQL
+      CREATE TYPE estimate_cost AS ENUM ('ground_transportation', 'lodging', 'meals', 'misc', 'registration', 'rental_vehicle', 'air', 'taxi', 'personal_auto', 'transit_other', 'train');
```
Comment (carolyncole) [inline db/migrate/20190501192018_add_type_to_estimates.rb]: Is cost the correct name here?  I would think that was a number.  cost_type maybe?

Comment (hackmastera) [reply db/migrate/20190501192018_add_type_to_estimates.rb]: `cost_type` sounds good. `type` alone made rails think I was trying to do single table inheritance 😁

**Comment by hackmastera** [comment]
I can move the attendance deletion to another PR.

**Comment by carolyncole** [comment]
I'm also noting circleci is cranky.  You may need `- run: sudo apt install -y postgresql-client || true` (from https://circleci.com/docs/2.0/postgres-config/)

**Comment by hackmastera** [comment]
@carolyncole yeah I've been working on the circleci / postgres problem for a while now. can't get `psql` to run. 🙁

**Comment by hackmastera** [comment]
@carolyncole okay it's clean and green please take another look when you can!

**Pull request merged**